specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-02-preview/SpringMiddlewares.json

```diff
@@ -0,0 +1,925 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-08-02-preview",
+    "title": "AppPlatformManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppPlatform/javaEnvironments/{javaEnvironmentName}/eurekaServers": {
+      "get": {
+        "tags": [
+          "EurekaServers"
+        ],
+        "description": "List the eureka server settings.",
+        "operationId": "EurekaServers_List",
```

Comment:
Here is duplicate with https://github.com/Azure/azure-rest-api-specs/blob/main/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-05-01-preview/appplatform.json#L967, please use an another name

same issues in this file are:
 EurekaServers_Get
EurekaServers_UpdatePatch
ConfigServers_Get
ConfigServers_UpdatePatch
ConfigServers_Validate